packages/console/src/pages/Settings/index.tsx

```diff
@@ -49,17 +48,21 @@ const Settings = () => {
   });
 
   return (
-    <Card className={classNames(detailsStyles.container, styles.container)}>
-      <CardTitle title="settings.title" subtitle="settings.description" />
-      <TabNav>
-        <TabNavItem href="/settings">{t('settings.tabs.general')}</TabNavItem>
-      </TabNav>
+    <div className={classNames(resourcesStyles.container, styles.container)}>
+      <div className={resourcesStyles.headline}>
+        <CardTitle title="settings.title" subtitle="settings.description" />
+      </div>
```

Comment:
Could we apply the headline styles directly to the CardTitle instead of wrap in a extra wrapper?

Reply:
updated.

Reply:
No, the CardTitle has its own structure and layout styles, if we apply the headline styles directly, the `<CardTitle />`'s looking will be wrong.

Reply:
The div wrapper is removed for it's unnecessary.